### ISSUE TITLE
Narrative Minor Feature Request

### ISSUE THREAD

**Issue by emmahodcroft** [nextstrain/auspice#883]
As I play with narratives today, just a couple of markdown features that would be handy if possible. If they are more trouble than they are worth, no worries. Just ideas!

I currently can't seem to get tables (either Markdown or HTML) working in the `auspiceMainDisplayMarkdown` part working. This might be nice for formatting lists of links with images, for example. Or lists of previous situation reports (or other languages)

I would like to give some links on the left-hand side to link to later slides:
![image](https://user-images.githubusercontent.com/14290674/73608543-c4730400-45c4-11ea-99e6-8d5fdfa9d495.png)

This currently works with 'hard' links (full URL links) but means the whole narrative is reloaded. This is probably fine, but it would be kinda cool if it was possible to just 'fast-forward' to the right slide. You can't link Markdown headers here, since that isn't how our URLs work (you just end up adding #coronviruses onto the end of the URL, like ...?n=1#coronaviruses). 

Both minor things, but if they were/are easy to implement, might be cooL!

**Comment by jameshadfield** [comment]
Agreed -- there are many little improvements such as this to be worked on.

> I currently can't seem to get tables (either Markdown or HTML) working in the auspiceMainDisplayMarkdown part working.

I presume HTML tables don't work because of https://github.com/nextstrain/auspice/blob/master/src/components/narrative/MainDisplayMarkdown.js#L136 (this code was copied from the footer markdown parsing). If markdown tables don't work then there may be an option we can add to https://github.com/nextstrain/auspice/blob/master/cli/server/parseNarrative.js#L13 to improve this.
